examples/randomwalks/randomwalks.py

```diff
@@ -1,106 +1,244 @@
+from typing import Dict, List, Optional, Tuple, Callable
+
 import networkx as nx
 import numpy as np
 import torch
 
 
-def randexclude(rng: np.random.RandomState, n: int, exclude: int) -> int:
+def generate_rand_int_excluding(
+    random_state: np.random.RandomState,
+    max: int,
+    exclude: int
+) -> int:
+    """Random integer generator, excluding a specific number
+
+    Args:
+        random_state: Numpy random state
```

Comment:
Rever naming. `rng` is a very popular var name for an `np.random.RandomState` instance.